herald/herald/Sensor/Analysis/VenueDiary.swift

```diff
@@ -285,23 +285,20 @@ public class VenueDiary: NSObject, SensorDelegate {
     }
     
     public func sensor(_ sensor: SensorType, didMeasure: Proximity, fromTarget: TargetIdentifier, withPayload: PayloadData) {
-        logger.debug("VenueDiary.sensor didMeasure withPayload")
         // parse payload if venue
         guard sensor == SensorType.BLE || sensor == SensorType.BEACON || sensor == SensorType.BLMESH else {
             // Not going to be a useful payload, so return
             return
         }
-        logger.debug("VenueDiary.sensor didMeasure withPayload - valid payload")
-        
         do {
             guard let encounter = try VenueEncounter(didMeasure, withPayload) else {
                 return
             }
-            // add encounter
+            logger.debug("didVisit(payload=\(withPayload.shortName))")
             let _ = addEncounter(encounter, with: withPayload, at: Date())
         } catch {
             // parse error - silently ignore but log
-            logger.fault("Error parsing Beacon payload: \(error)")
+            //logger.fault("Error parsing Beacon payload: \(error)")
```

Comment:
Please reinstate this log line

Reply:
Reinstated. Only disabling logging for VenuePayloadParseError.UnsupportedPayloadIdAndVersion as that is a common condition, e.g. payload is Herald phone, not beacon.